### PR TITLE
Support multipath device

### PR DESCRIPTION
**Pull request by zengyingzhe**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: contrib/connector/fc/linuxfc.go [hunk 1]
```diff
@@ -176,3 +176,28 @@ func trimDoubleQuotesInText(str string) string {
 	}
 	return str
 }
+
+func getMultipathDevice(deviceWWN string) (string, error) {
+	cmd := fmt.Sprintf("ls -l /dev/disk/by-id/ | grep %s", deviceWWN)
+	out, err := connector.ExecCmd("/bin/bash", "-c", cmd)
+	if err != nil {
+		errMsg := fmt.Sprintf("Error occurred when find DM of wwn %s: %s, %v", deviceWWN, out, err)
```
Comment (ThisIsClark) [inline contrib/connector/fc/linuxfc.go]: Error strings should not be capitalized.
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Comment (ThisIsClark) [reply contrib/connector/fc/linuxfc.go]: Same as other error strings

**Comment by wisererik** [comment]
Hi @zengyingzhe is the terminateConnection fine, I found one issue here, please handle it in thir pr
image

**Review by wisererik — APPROVED** [review]
LGTM

**Pull request merged**